server/index.js

```diff
@@ -19,7 +19,7 @@ app.use(bodyParser.json());
 
 // Handle GET request for index.html
 app.get('/:propertyId', (req, res) => {
-  res.sendfile(path.resolve(__dirname + '/../public/dist/index.html'));
+  res.sendFile(path.resolve(__dirname + '/../public/dist/index.html'));
 });
```

Comment:
you can actually remove this function entirely by replacing line 17 with `app.use('/:propertyId', serve);`